container_checkpoint_test.go

```diff
@@ -408,3 +415,126 @@ func TestCheckpointLeaveRunning(t *testing.T) {
 
 	<-statusC
 }
+
+func withCheckpointOpts(rt, path string) CheckpointTaskOpts {
```

Comment:
This would also be nice as a package opt for other clients.  Maybe something like `WithCheckpointImagePath`.

Reply:
Move `withCheckpointOpts` and `withRestoreOpts` into `task_opts.go`, since they are taskOpts